src/main/java/seedu/mindmymoney/data/Lists.java

```diff
@@ -8,5 +8,67 @@
  * Container for lists.
  */
 public class Lists {
-    public static final ArrayList<Expenditure> expenditures = new ArrayList<>();
+    public ArrayList<Expenditure> itemList;
```

Comment:
Consider setting `itemList` to private, and exposing a getter method for better encapsulation?